consensus/src/main/kotlin/maru/consensus/qbft/adapters/ForksScheduleAdapter.kt

```diff
@@ -0,0 +1,73 @@
+/*
+   Copyright 2025 Consensys Software Inc.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+ */
+package maru.consensus.qbft.adapters
+
+import java.math.BigInteger
+import java.util.Optional
+import maru.config.QbftOptions
+import maru.consensus.ForkSpec
+import maru.consensus.qbft.QbftConsensusConfig
+import org.apache.tuweni.bytes.Bytes
+import org.hyperledger.besu.config.BftConfigOptions
+import org.hyperledger.besu.datatypes.Address
+import org.hyperledger.besu.consensus.common.ForkSpec as BesuForkSpec
+import org.hyperledger.besu.consensus.common.ForksSchedule as BesuForksSchedule
+
+class ForksScheduleAdapter(
+  currentSpec: ForkSpec,
+  config: QbftOptions,
+) : BesuForksSchedule<BftConfigOptions>(maruForkSpecsToBesu(currentSpec, config)) {
+  companion object {
+    fun maruForkSpecsToBesu(
+      currentSpec: ForkSpec,
+      config: QbftOptions,
+    ): MutableCollection<BesuForkSpec<BftConfigOptions>> =
+      mutableListOf(BesuForkSpec(0, createBftConfig(currentSpec, config)))
```

Comment:
This isn't creating a `ForkSchedule` as it uses the same values for every block height. Ok with doing this in another PR as will probably need to make changes to the `BlockTimer` so it can use timestamps as well as block numbers like the `ProtocolSchedule` in Besu does.